dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/common/RuntimeUtil.java

```diff
@@ -83,4 +83,20 @@ public static void loadProperties(Properties properties, File file) throws IOExc
         }
     }
 
+    public static String getDependencies(Properties properties) {
+        String deps = properties.getProperty("camel.jbang.dependencies");
```

Comment:
Warning `properties` could be null according to https://github.com/apache/camel/pull/8749/files#diff-b62a4b0acb50014174399c284b5540cf84016571e5c789a23edc8d843692721cL345

Reply:
To be clear, if `properties` is `null`, calling `properties.getProperty("camel.jbang.dependencies")` will throw a `NullPointerException`

Reply:
My understanding is that properties are expected to be not null, if that's not the case we can see NullPointerException at so many other places.

Reply:
If you check these lines https://github.com/apache/camel/blob/59c6450582fa662c168021360c42f1e78a4cc438/dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/Run.java#L266-L269, you will see that `profileProperties` is `null` if the file doesn't exist which can occur so I confirm that `properties` can be `null`

Reply:
Sure, I should have the fix soon.